microcosm_flask/paging.py

```diff
@@ -47,21 +49,27 @@ def from_query_string(cls, qs):
         This dictionary should probably come from `PageSchema.from_request()`.
 
         """
+        dct = qs.copy()
+        offset = dct.pop("offset")
+        limit = dct.pop("limit")
         return cls(
-            offset=qs["offset"],
-            limit=qs["limit"],
+            offset=offset,
+            limit=limit,
+            **dct
```

Comment:
Every `Page` now has access to the `rest` of the query string arguments. By default, these will be shown in the paging URIs. (To omit, use a narrower paging query or sub-class `Page`)